### ISSUE TITLE
Events lost in /blocks 

### ISSUE THREAD

**Issue by Ljzn** [paritytech/substrate-api-sidecar#290]
**Description**

Some evnets lost in the response of
`blocks/0x03340cd4add28b94760bec782585b36f6ac2b38d0f7dcbb7049c6e971fd2e374`

```js
{
    number: "1622398",
        hash: "0x03340cd4add28b94760bec782585b36f6ac2b38d0f7dcbb7049c6e971fd2e374",
            parentHash: "0xe7fb99a29e4d27dacd5d87a85d4e862767f07a6d9318947da238c7fc23022575",
                stateRoot: "0xe727ec576740f63d70b2f9723cb3f1bf3acc4f60abc5bc6656039b1acb1fdff6",
                    extrinsicsRoot: "0x57e1b7e5501ff23be5c2ba35494bbebb3d4ac0e89f4cd59e878ad24780a279a9",
                        authorId: "14bJ3GACqzyN3oUvCCbhWKMhbLnCsxaxo7efYg2rUkq8MHPL",
                            logs: [
                                {
                                    type: "PreRuntime",
                                    index: "6",
                                    value: [
                                        "BABE",
                                        "0x015b0000009ab7e50f000000000c48edee0508ceab422914b5ad5ac34edd3f06af9f3328e576e137fc205cfd16a1c205ce7d979c42fa92f29386b404807b582bf1acd6011f4dd1fcacf412380442d509956599307d8bfd967d4b8a940a6bec591034ce8bebb82b144f8d5abf0f"
                                    ]
                                },
                                {
                                    type: "Seal",
                                    index: "5",
                                    value: [
                                        "BABE",
                                        "0xb6a00ae72137bf3676c516a5c7136f5f7cae9d96fd4ba970facd5ff586dda25dcbd235339d0588dc04b23cb40178ddbca7510f261f1b4c7827763c3ff1e9a987"
                                    ]
                                }
                            ],
                                onInitialize: {
        events: []
    },
    extrinsics: [
        {
            method: {
                pallet: "timestamp",
                method: "set"
            },
            signature: null,
            nonce: "0",
            args: {
                now: "1600277916000"
            },
            tip: "0",
            hash: "0xa6717191501c4af1026132defe8f128d73f9192eef4f0cd775ec9c3c06088d79",
            info: {},
            events: [
                {
                    method: {
                        pallet: "system",
                        method: "ExtrinsicSuccess"
                    },
                    data: [
                        {
                            weight: "159133000",
                            class: "Mandatory",
                            paysFee: "Yes"
                        }
                    ]
                },
                {
                    method: {},
                    data: null
                },
                {
                    method: {},
                    data: null
                },
                {
                    method: {},
                    data: null
                },
                {
                    method: {},
                    data: null
                },
                {
                    method: {},
                    data: null
                },
                {
                    method: {},
                    data: null
                },
                {
                    method: {},
                    data: null
                }
            ],
            success: true,
            paysFee: true
        },
        {
            method: {
                pallet: "claims",
                method: "claimAttest"
            },
            signature: null,
            nonce: "0",
            args: {
                dest: "12Sz3YJih3NFjFUYGeGB5LnqrZNyytEpWaoBwP5tMSHuxaAV",
                ethereum_signature: "0xfc90d5e91ac8dfae539787fd046962bfb8f275b44090508e003b390d3fd3b5830e4480bf7c2aa776d94b133f9001446fd24871099443646c4fa834892597ae0301",
                statement: "0x492068657265627920616772656520746f20746865207465726d73206f66207468652073746174656d656e742077686f7365205348412d323536206d756c74696861736820697320516d63315859715436533339574e70325565695255725a6963685557555070474554684445366441623366364e792e202854686973206d617920626520666f756e64206174207468652055524c3a2068747470733a2f2f73746174656d656e742e706f6c6b61646f742e6e6574776f726b2f726567756c61722e68746d6c29"
            },
            tip: "0",
            hash: "0x0ca2a632f03a0dde7add1f0d0bbf4ebb54c8ce231d82328d4d7af9eb7c55d596",
            info: {},
            events: [
                {
                    method: {
                        pallet: "system",
                        method: "NewAccount"
                    },
                    data: [
                        "12Sz3YJih3NFjFUYGeGB5LnqrZNyytEpWaoBwP5tMSHuxaAV"
                    ]
                },
                {
                    method: {
                        pallet: "balances",
                        method: "Endowed"
                    },
                    data: [
                        "12Sz3YJih3NFjFUYGeGB5LnqrZNyytEpWaoBwP5tMSHuxaAV",
                        "73409000000000"
                    ]
                },
                {
                    method: {
                        pallet: "slots",
                        method: "NewLeasePeriod"
                    },
                    data: [
                        "593825088"
                    ]
                }
            ],
            success: false,
            paysFee: null
        },
        {
            method: {
                pallet: "balances",
                method: "transfer"
            },
            signature: {
                signature: "0x9a67e0267692eb93a2c5f3878fe818745ffe9f7b2ffbb7b85c092b2250896e1a2b8366f0485c9026edb03d73369fff0938f89e0a6cc7649551329dbdd22f8389",
                signer: "12pkT3YQXktz3Q6n4mCzkhZ3oSCsDu2E4saQ8nKcVXgh6Sgx"
            },
            nonce: "0",
            args: {
                dest: "12xtAYsRUrmbniiWQqJtECiBQrMn8AypQcXhnQAc6RB6XkLW",
                value: "99836000000"
            },
            tip: "0",
            hash: "0x38b4795451c71e77771f1cb8e8447b20c037e882df8020c9c49e705fcbceed8f",
            info: {},
            events: [],
            success: false,
            paysFee: null
        }
    ],
        onFinalize: {
        events: []
    }
}
```

**Steps to Reproduce**

sidecar version: 0.17.0,
polkadot version: 0.8.23

**Expected vs. Actual Behavior**

Get the same data as https://polkascan.io/polkadot/block/1622398

**Comment by emostov** [comment]
Thank you for filing the issue.

I was able to replicate this issue on master (commit: 709bbb4) and v0.17.0 by querying `/blocks/1` and then `/blocks/1622398`. However if I start Sidecar and go straight to `/blocks/1622398` it works correctly.

 It looks like the fix for this https://github.com/polkadot-js/api/issues/2613 might not fully helping changing out types.

I will look into this further. 

cc @dvdplm, @joepetrowski, @jacogr

**Comment by emostov** [comment]
Closed by #291
